projects/wonka-ui/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sintef/wonka-ui",
-  "version": "1.1.0",
+  "version": "1.1.1-beta.0",
```

Comment:
This file is auto-generated but it would be cleaner to push the one that is coherent with package.json

Reply:
Indeed, I'm fixing it now!

Good point, will update it.

Reply:
Still not changed.

Reply:
Fixed now!